openTrivia/getQuestions.js

```diff
@@ -0,0 +1,21 @@
+//問題の取得
+getQuestions = () => {
+
+    fetch('https://opentdb.com/api.php?amount=10&type=multiple')
+        .then((response) => {
+            if(response.ok) {
+                return response.json();
+            } else {
+                throw new Error();
+            }
+        })
+        .then((json) => {
+
+            //jsonでデータセット
+            dataQuestion = json['results'];
+
+            //インスタンス生成（取得する問題の要素番号）
+            createQuestion(0);
+        })
+        .catch((error) => console.log(error));
+}
```

Comment:
`async` `await` を使うとこんな感じにすっきり書けます。
```suggestion
getQuestions = async () => {

  try {
    const response = await fetch('https://opentdb.com/api.php?amount=10&type=multiple');
    const json = await response.json();
    //jsonでデータセット
    dataQuestion = json['results'];
    //インスタンス生成（取得する問題の要素番号）
    createQuestion(0);
  } catch (e) {
    console.log(e);
  }
}
```